src/content/manifest/overview-manifest.mdx

```diff
@@ -0,0 +1,153 @@
+---
+title: Okteto Manifest
+description: An overview of the Okteto Manifest and its core features.
+sidebar_label: Overview
+id: overview-manifest
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The Okteto manifest is how you configure the behavior for building, deploying, and developing your application in an Okteto development environment. If you already have a Docker Compose file for your application, then the Okteto manifest is optional. 
+
+There are five sections of the Okteto manifest that define how your development environments and containers behave.
+
+| Section | Description | Reference |
+| ------- | ----------- | --------- |
+| [build](#build) | Configure a list of images to build for your application's development environment. | [Learn more](reference/okteto-manifest.mdx#build-object-optional) |
+| [deploy](#deploy) | Configure a list of commands to execute within the development environment to deploy your application. | [Learn more](reference/okteto-manifest.mdx#deploy-string-optional) |
+| [destroy](#destroy) | Configure a list of commands to destroy all resources created in the `deploy` and `external` sections. | [Learn more](reference/okteto-manifest.mdx#destroy-string-optional) |
+| [dev](#dev) | Configure a list of development containers and their behavior to define how `okteto up` works when you start working on a specific, configured service in your development environment. | [Learn more](reference/okteto-manifest.mdx#dev-object-optional) |
+| [external](#external) | Configure a list of external resources used by your application's development environment. These are resources that exist outside of the Okteto cluster (e.g. cloud resources, dashboards). | [Learn more](reference/okteto-manifest.mdx#external-object-optional). |
+
+There are other sections to help you build your ideal development experience in Okteto, but this document focuses solely on introducing you to the core concepts of the Okteto manifest. You can learn more about the other Okteto manifest options in the [manifest reference](reference/okteto-manifest.mdx).
+
+## Build
```

Comment:
I would have a dedicated doc for each section, as we already do for `Variables` and `External Resources`. I like the quick introduction here, but we should add a link to the detailed page when it's available

Reply:
Agreed. That's the goal. So is the feedback here basically "Let's make sure to circle back and add a link to the more detailed page once it's available"?